### DISCUSSION TITLE
Grafana Plugin how to get visualization type

### DISCUSSION THREAD

**Discussion by Jeric-Disney** [grafana/grafana#54140] [Dashboards, Panels & Visualizations]
I am creating a grafana plugin and I need to get the visualization type in the datasource. How to get the selected visualization type? table, time series, log, bar chart, stat, etc.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
